storage/memory/map.go

```diff
@@ -8,8 +8,11 @@ import (
 	"github.com/ipfs/go-cid"
 )
 
-// Storage is in-memory implementation of content storage. Should be used only for testing
+func New() *Storage {
```

Comment:
Nit: Naming it like `NewStorage` would be better to avoid confusion with other constructors in future.